frameworks/Java/activeweb/benchmark_config

```diff
@@ -4,40 +4,23 @@
     "default": {
       "setup_file": "setup",
       "json_url": "/activeweb/json",
-      "plaintext_url": "/activeweb/plaintext",
-      "port": 8080,
-      "approach": "Realistic",
-      "classification": "Platform",
-      "database": "None",
-      "framework": "activeweb",
-      "language": "Java",
-      "orm": "Raw",
-      "platform": "ActiveWeb",
-      "webserver": "Resin",
-      "os": "Linux",
-      "database_os": "Linux",
-      "display_name": "ActiveWeb",
-      "notes": "",
-      "versus": "servlet"
-    },
-    "raw": {
-      "setup_file": "setup",
       "db_url": "/activeweb/db",
       "query_url": "/activeweb/queries?queries=",
       "fortune_url": "/activeweb/fortunes",
       "update_url": "/activeweb/updates?queries=",
+      "plaintext_url": "/activeweb/plaintext",
       "port": 8080,
       "approach": "Realistic",
-      "classification": "Platform",
+      "classification": "Full-stack",
```

Comment:
I think specifying `servlet` as the platform is appropriate. We have Gemini listed as a "Fullstack" framework running on the "Servlet" platform.

Classifying something as a "Platform" is more like `Undertow` - while it implements all the tests, it is really not a framework per se.